source/_components/media_player.androidtv.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: "Android TV"
+description: "Documentation on how to integrate any Android TV / Android device into Home-Assistant"
+date: 2018-12-10 01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+ha_release: 0.85
+ha_iot_class: "Local Polling"
+---
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  # A device that does not require ADB authentication, or uses the generated ADB keys's default location
+  - platform: androidtv
+    host: 192.168.1.37
+    name: MIBOX3
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.